src/euphorie/deployment/locales/nl/LC_MESSAGES/euphorie.po

```diff
@@ -2243,6 +2243,11 @@ msgstr ""
 msgid "description_measures_text_handling"
 msgstr ""
 
+#. Default: "Find an OiRA consultant you want to contact or add to your organisation to provide support."
+#: Euphorie/src/euphorie/client/browser/templates/consultants.pt:44
+msgid "description_oira_consultants"
+msgstr "Ontdek hoe u in contact kunt komen met een OiRA-adviseur op ${email}."
```

Comment:
I‌ think `${email}` is wrong here. The original msgstr doesn't conain a `i18n:name` part.